src/pages/instances/actions/MigrateInstanceBtn.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from "react";
+import { Button } from "@canonical/react-components";
+import MigrateInstanceForm from "pages/instances/MigrateInstanceForm";
+import usePortal from "react-useportal";
+import { migrateInstance } from "api/instances";
+import { failure, info, success } from "context/notify";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchClusterMembers } from "api/cluster";
+import Loader from "components/Loader";
+import { useEventQueue } from "context/eventQueue";
+import { Notification } from "types/notification";
+
+interface Props {
+  instance: string;
+  project: string;
+  setInTabNotification: (msg: Notification) => void;
+}
+
+const MigrateInstanceBtn: FC<Props> = ({
+  instance,
+  project,
+  setInTabNotification,
+}) => {
+  const eventQueue = useEventQueue();
+  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+  const queryClient = useQueryClient();
+
+  const { data: members = [], isLoading } = useQuery({
+    queryKey: [queryKeys.cluster, queryKeys.members],
+    queryFn: fetchClusterMembers,
+  });
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  const handleSuccess = () => {
+    setInTabNotification(
+      success(`Migration finished for instance ${instance}`)
```

Comment:
I think we should use the `ItemName` component, with the `bold` prop, here and in all the other notifications where we mention the instance name.

Reply:
In the failures, the name is part of the title which is bold itself, but here we can use it, sure.